test/templates/helpers/cleanJsString.js

```diff
@@ -0,0 +1,29 @@
+var Lab = require('lab');
+var Code = require('code');
+
+var cleanJsString = require('../../../templates/helpers/cleanJsString');
+
+var lab = exports.lab = Lab.script();
+
+lab.experiment('Template Helper cleanJsString', function () {
+  lab.test('Test jsesc', function (done) {
+    Code.expect(cleanJsString('Ich ♥ Bücher'), 'Ich \\u2665 B\\xFCcher');
+
+    done();
+  });
+
+  lab.test('Remove <script>', function (done) {
+    Code.expect(cleanJsString('<script>var i=0;</script>'), 'var i=0;');
+    Code.expect(cleanJsString('<script>var i=0;'), 'var i=0;');
+    Code.expect(cleanJsString('var i=0;</script>'), 'var i=0;');
```

Comment:
Where does `cleanJsString` remove `<script>`? It should only replace `</script` with `<\/script`.